### ISSUE TITLE
Configure model hyperparameters, such as input size, number of epochs, learning rate, etc.

### ISSUE THREAD

**Issue by otman-dev** [AI-ForestFireDetection/AI-Yolo#19]
Related to UserStory #5
